app/code/Magento/Integration/Helper/Oauth/Data.php

```diff
@@ -116,22 +116,22 @@ public function getConsumerPostTimeout()
     /**
      * Get customer token lifetime from config.
      *
-     * @return int hours
+     * @return string hours
```

Comment:
Why docblock return type changed to string? According to implementation, it should be numeric.

Reply:
Comment updated